backend/.env.example

```
@@ -1,3 +1,8 @@
 
-PORT=
-MONGODB_URL=
\ No newline at end of file
+PORT=3000
+MONGODB_URL=mongodb://localhost:27017/shopsmart
+JWT_SECRET=secret
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The JWT_SECRET key should go before the MONGODB_URL key

(UnorderedKey)